overlays/bdc-overlay.yaml

```diff
@@ -0,0 +1,13 @@
+machines:
+  '0':
+    constraints: cores=2 mem=4G root-disk=16G
+    series: bionic
+  '1':
+    constraints: cores=8 mem=64G root-disk=100G
+    series: bionic
```

Comment:
Is specifying the series here necessary?  Shouldn't it already be bionic, unless / until we want to switch to focal anyway, in which case this would revert that?

Reply:
ditto